### DISCUSSION TITLE
Resolution of some website details

### DISCUSSION THREAD

**Discussion by joseph-mueller** [IviFoundation/ivifoundation.github.io#83] [General]
I'd like to use this discussion to get general feedback on the website.  Here are some specific questions:

1. Do people like the general structure we have now?  (ignore the naming for this item).  Right now we have:
    - Root level stuff (roots of hierarchy, privacy policy, how to join,....)
    - About -- this has technical information about our standards.   I would like to see more on VISA, HiSLIP, and SCPI.  But there is some
    - Shared Components-- directly mimics the existing site.  With Shared Components at the root and some old stuff underneath
    - Specification -- A root level page with links for the specs.   We might want to break this into multiple pages under a hierarchy
    - About the IVI Foundation -- Has information about the organization
    - Driver Registry -- I think another parent page that talks about the purpose of the registry with a child showing the registry and another child to make an entry in the registry.

2. I'm mildly incline to put _Privacy Policy_ & _How to Join_  under _About the IVI Foundation_.   What do you think?
3. I would like to delete the page "Committees"  I think it is fully covered by _About the IVI Foundation_
4. I would like to  setup a blog for meeting announcements and get rid of the last vestiges of old meetings (maybe populate some old wikis)
5. I would like a paragraph on the BoD under Operation.
6. We need to clean up the current home page (direct clone from existing site).   I think it may belong in a blog entry.
7. I'd like to delete the oldfiles directory 

Finally, I need to do some work on the frame, but I think I can propose something based on our discussion last week.

**Comment by alejandro5042** [comment]
> Do people like the general structure we have now? (ignore the naming for this item). Right now we have [...]

- _About the IVI Foundation_ should be renamed to _About_ (it's simpler and convention that "about" is about the org)
- _About_ renamed to _Standards_, as per [World Wide Web Consortium (W3C)](https://www.w3.org/)
- Root level pages:
	- Standards
	- Downloads? (includes SC and Specs list)
- General discussion around IVI being an org that facilitates HW instruments, not just about drivers
- But really:
	- Let's organize the content with common sense and we can adjust later

> I'm mildly incline to put Privacy Policy & How to Join under About the IVI Foundation. What do you think?

- Leave it at the bottom in the frame, no need to highlight at the top-level nav
- How to join -- put it under About the IVI Foundation; or move Current Members and How to Join under a new Membership top-level

> I would like to delete the page "Committees" I think it is fully covered by About the IVI Foundation

- It does not appear to have valuable content, and this content is duplicated in other places, let's delete

> I would like to setup a blog for meeting announcements and get rid of the last vestiges of old meetings (maybe populate some old wikis)

- Already covered by #20 
- Create an Announcements blog to cover:
  - Meeting announcements
  - New specs
- Blog is a top-level link

> I would like a paragraph on the BoD under Operation.

- Yes, let's do it

> We need to clean up the current home page (direct clone from existing site). I think it may belong in a blog entry.

- Security section --> make it a blog post
- VXIplug&play/SCPI --> this is a category under Standards
- Change the title of the home page to just "Home"

For example:

![image](https://user-images.githubusercontent.com/3143819/236323387-fcf0fcd8-c195-4e4c-a996-43f291f55fbd.png)

```html
<div style="background: #ddf;padding: 10px 20px;border-radius: 1em;"><h2 id="getting-started" style="
    margin-top: 0;
"> <a href="#getting-started" class="anchor-heading" aria-labelledby="getting-started"><svg viewBox="0 0 16 16" aria-hidden="true"><use xlink:href="#svg-link"></use></svg></a> Getting Started</h2><p>For general information on getting started using IVI drivers, see the <a href="/downloads/IVI-GSG-CurrentVersion.pdf">IVI Getting Started Guide (PDF)</a>. For more information on the IVI standards, see <a href="resources/Default.html">More Information</a>.</p></div>
```

> I'd like to delete the oldfiles directory

- Yes
- Solve the size issue later, when we deal with binaries

**Reply by alejandro5042** [reply]
@joseph-mueller Finally! I think I got rate limited or something? Couldn't post until just now... weird.
